src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -138,6 +138,10 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, ControlValue
   /** Type of the search input field */
   @Input() type = 'text';
 
+  /** Icon / svg used for displaying Close-Icon */
```

Comment:
Please add two separate docs to explain that the svg will override the standard one
Please also add them to https://github.com/bithost-gmbh/ngx-mat-select-search#inputs